mac/build.sh

```diff
@@ -153,7 +155,7 @@ while [[ $# -gt 0 ]] ; do
             ;;
         -deploy-only)
             SKIP_BUILD=true
-            shift # past argument
+            #shift # past argument
```

Comment:
Is this safe?  Or was this a temp edit that slipped in?

Reply:
This is safe... it shouldn't have had a `shift` in there so I should probably remove the line altogether.